src/Kiota.Builder/Writers/TypeScript/CodeFileBlockEndWriter.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Kiota.Builder.CodeDOM;
+
+namespace Kiota.Builder.Writers.TypeScript;
+
+public class CodeFileBlockEndWriter : ICodeElementWriter<CodeFileBlockEnd>
+{
+    public void WriteCodeElement(CodeFileBlockEnd codeElement, LanguageWriter writer)
```

Comment:
If this does nothing, should we consider removing it and not registering in the list of writers?